mkdocs.yml

```diff
@@ -67,112 +73,142 @@ extra_javascript:
+      - Available Wallets: "evm/wallets.md"
+      - Ethereum Compatibility:
+          - Account System: "evm/ethereum-compatibility/account-system.md"
+          - JSON-RPC APIs: "evm/ethereum-compatibility/jsonrpc.md"
+          - Gas Compatibility: "evm/ethereum-compatibility/gas.md"
+          - EVM Tracing: "evm/ethereum-compatibility/evm-tracing.md"
           - Precompiled Contracts:
-              - Overview: 'evm/ethereum-compatibility/precompiles/overview.md'
-              - Ethereum: 'evm/ethereum-compatibility/precompiles/ethereum.md'
-              - Staking: 'evm/ethereum-compatibility/precompiles/staking.md'
-              - Deposit: 'evm/ethereum-compatibility/precompiles/deposit.md'
-              - Commitment Token: 'evm/ethereum-compatibility/precompiles/commitment-token.md'
-              - USDT: 'evm/ethereum-compatibility/precompiles/usdt.md'
-              - StateStorage: 'evm/ethereum-compatibility/precompiles/state-storage.md'
-              - Dispatch: 'evm/ethereum-compatibility/precompiles/dispatch.md'
-              - Conviction Voting: 'evm/ethereum-compatibility/precompiles/conviction-voting.md'
-    - Darwinia Staking:
-          - Commitment Token: 'evm/darwinia-staking/commitment-token.md'
-          - Staking Module: 'evm/darwinia-staking/staking.md'
-          - Deposit Module: 'evm/darwinia-staking/deposit.md'
-    - Governance: 'evm/governance.md'
-    - Tutorial:
-          - Transfer Token Guide: 'evm/tutorial/transfer-token.md'
-          - Staking Guide: 'evm/tutorial/staking.md'
-          - USDT Guide: 'evm/tutorial/usdt.md'
-          - Multisig Wallet: 'evm/tutorial/multisig-wallet.md'
-          - Create DAO Guide: 'evm/tutorial/create-dao.md'
-          - Smart Contract Guide:
-              - Interact With Web3.js: 'evm/tutorial/smart-contract/interact-with-web3js.md'
-              - Interact With Ethers.js: 'evm/tutorial/smart-contract/interact-with-ethersjs.md'
-              - Interact With Hardhat: 'evm/tutorial/smart-contract/interact-with-hardhat.md'
-              - Interact With Foundry: 'evm/tutorial/smart-contract/interact-with-foundry.md'
-              - Verify Smart Contract: 'evm/tutorial/smart-contract/verify-contract.md'
-              - Awesome Tutorial: 'evm/tutorial/smart-contract/awesome-tutorial.md'
-          - Chain Guide:
-              - Chain Database Snapshot: 'evm/tutorial/chain/snapshot.md'
-              - Run Collator Node: 'evm/tutorial/chain/run-collator-node.md'
-              - Run Archive Node: 'evm/tutorial/chain/run-archive-node.md'
-              - Run EVM Tracing Node: 'evm/tutorial/chain/run-evm-tracing-node.md'
-              - Run Dev Node: 'evm/tutorial/chain/run-dev-node.md'
-              - Run Relay Chain Testnet: 'evm/tutorial/chain/run-relay-chain-testnet.md'
+              - Overview: "evm/ethereum-compatibility/precompiles/overview.md"
+              - Ethereum Contracts: "evm/ethereum-compatibility/precompiles/ethereum.md"
```

Comment:
No need to add `Contracts`, there is already one in the folder `Precompiled Contracts`.

Reply:
I prefer:
```suggestion
              - Ethereum Precompile: "evm/ethereum-compatibility/precompiles/ethereum.md"
```